examples/nodejs-simple/README.md

```diff
@@ -1,4 +1,109 @@
-### Instructions
-`npm install` - install dependencies
-
-`npm start` - start the self-running example
+# Simple node.js Example
+
+This is a very simple "server" that doesn't do much other than show how the SDK works in node.js.
+
+It will
+- Setup the Agones SDK
+- Call `sdk.ready()` to register that it is ready with Agones.
+- Every 10 seconds, write a log showing how long it has been running for
+- After 60 seconds, call `sdk.shutdown()` to shut the server down.
+
+To learn how to deploy this example service to GKE, please see the tutorial [Build and Run a Simple Gameserver (node.js)](https://agones.dev/site/docs/tutorials/simple-gameserver-nodejs/).
+
+## Building
+
+If you want to modify the source code and/or build an updated container image, run `make build` from this directory.
+This will run the `docker build` command with the correct context.
+
+This example uses the a Docker container to host the SDK and example it inside a container so that no special build
+tools need to be installed on your system.
+
+## Running locally with Docker
```

Comment:
```suggestion
## Testing locally with Docker
```

Suggestion. Don't need to change.

Reply:
That sounds a bit better (and I also saw a typo in the paragraph above that I fixed too).